docs/src/overview/comparison.md

```diff
@@ -0,0 +1,58 @@
+# Comparison with commercial Solutions
+
+As metal-stack is the foundation to build kubernetes clusters on premise on bare metal, there are several commercial solutions available which offer management of kubernetes.
+In this document we describe the difference of some of the most popular solutions. It´s is not a complete list.
+
+```@contents
+Pages = ["comparison.md"]
+Depth = 5
+```
+
+In this document we want to compare a often cited battle between Kubernetes managed by Gardener on Metal Stack and Openshift running on VMWare.
```

Comment:
This sentence at least feels redundant because it starts with the same beginning "In this document we" as last sentence.